solidity/random-beacon/package.json

```diff
@@ -46,7 +46,7 @@
     "@nomiclabs/hardhat-etherscan": "^3.1.0",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
     "@openzeppelin/hardhat-upgrades": "^1.20.0",
-    "@tenderly/hardhat-tenderly": "^1.0.12",
+    "@tenderly/hardhat-tenderly": "1.0.12",
```

Comment:
Shouldn't we set range `>=1.0.13 <1.2.0` as in `ecdsa` instead of a fixed value?

Reply:
nvm here is an explanation f8f74ca